src/sources/multisrc/madara/MadaraScraper.js

```diff
@@ -79,11 +87,12 @@ class MadaraScraper {
     novel.novelName = loadedCheerio('.post-title h1').text().trim();
 
     novel.novelCover =
+      loadedCheerio('.summary_image > a > img').attr('data-lazy-src') ||
       loadedCheerio('.summary_image > a > img').attr('data-src') ||
       loadedCheerio('.summary_image > a > img').attr('src') ||
       defaultCoverUri;
 
-    loadedCheerio('.post-content_item', '.post-content').each(function () {
+    loadedCheerio('.post-content_item' || '.post-content').each(function () {
```

Comment:
Wouldn't `'.post-content_item'` always be true and it'd never go to the second condition. 

Also is the previous code working as I think loadedCheerio takes a selector as input?

```ts
loadedCheerio('.post-content_item', '.post-content').each(function () {

Reply:
I figured out how commas are supposed to work. Now it properly obtains the union of the two selectors as defined here https://webplatform.github.io/docs/css/selectors/